modules/aws-backup-source/backup_plan.tf

```
@@ -122,6 +122,40 @@ resource "aws_backup_plan" "aurora" {
   }
 }
 
+
+resource "aws_backup_plan" "parameter_store" {
+  count = var.backup_plan_config_parameter_store.enable ? 1 : 0
+  name = "${local.resource_name_prefix}-parameter-store-plan"
+
+  dynamic "rule" {
+    for_each = var.backup_plan_config_parameter_store.rules
+    content {
+      recovery_point_tags = {
+        backup_rule_name = rule.value.name
+      }
+      rule_name                = rule.value.name
+      target_vault_name        = aws_backup_vault.main.name
+      schedule                 = rule.value.schedule
+      completion_window        = rule.value.completion_window
+      enable_continuous_backup = rule.value.enable_continuous_backup != null ? rule.value.enable_continuous_backup : null
+      lifecycle {
+        delete_after       = rule.value.lifecycle.delete_after != null ? rule.value.lifecycle.delete_after : null
+        cold_storage_after = rule.value.lifecycle.cold_storage_after != null ? rule.value.lifecycle.cold_storage_after : null
```

Is there a reason we need these conditionals?  Can it just be this:


```suggestion
        delete_after       = rule.value.lifecycle.delete_after
        cold_storage_after = rule.value.lifecycle.cold_storage_after
```

It feels really weird to say "if it's not null, use it, but if it's null, use null"